test/images_test.py

```diff
@@ -48,6 +50,26 @@ def test_imagecube_grad(coords):
     # https://github.com/pytorch/pytorch/issues/50014
     loss.backward()
 
+# test for proper fits scale
+def test_imagecube_tofits(coords):
```

Comment:
You might try using the "tmp_path" parameter provided as part of Pytest, and then saving the FITS file to this temporary directory (e.g., https://github.com/MPoL-dev/MPoL/blob/main/test/images_test.py#L108)

The tmp_path is a [Pytest fixture](https://docs.pytest.org/en/6.2.x/fixture.html) that creates a temporary directory for the duration of the run. That way, when the tests are complete, the directory is automatically cleaned up.